Scripts/xbbtools/nextorf.py

```diff
@@ -171,17 +178,18 @@ def GetCDS(self, seq, strand=1):
                 elif codon_type == STOP:
                     if start_site == 0:
                         continue
-#                    if codon == 'XXX': print('do something')
+                    #                    if codon == 'XXX': print('do something')
                     stop = pos + 2
-#                    print("stop")
+                    #                    print("stop")
                     length = stop - start_site + 1
```

Comment:
I think the comments should be closer to the hash as "# comment..."

Reply:
This does look odd - and yes, worth a manual fix.

I think black assumed the lines would be: indentation, hash, space, code - but here they were hash, indentation, code.